### PR TITLE
SONAR-23309 Update Helm chart dependencies

### PR DESCRIPTION
**Pull request by carminevassallo**

### CODE REVIEW

File: .cirrus/setup.sh [hunk 1]
```diff
@@ -2,26 +2,24 @@
...
-export PATH=${BASE_FOLDER}/bin:${BASE_FOLDER}/google-cloud-sdk/bin:${PATH}
-
 mkdir -p ${BASE_FOLDER}
 
 curl -LO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-${GCLOUD_CLI_VERSION}-linux-x86_64.tar.gz
 echo "${GCLOUD_CLI_CHECKSUM}  google-cloud-cli-${GCLOUD_CLI_VERSION}-linux-x86_64.tar.gz" | sha256sum -c
 tar -xf google-cloud-cli-${GCLOUD_CLI_VERSION}-linux-x86_64.tar.gz
 chmod +x ./google-cloud-sdk
 mv ./google-cloud-sdk ${BASE_FOLDER}/google-cloud-sdk
-rm -rf google-cloud-cli-${GCLOUD_CLI_VERSION}-linux-x86_64.tar.gz
```
Comment (jCOTINEAU) [inline .cirrus/setup.sh:18]: i think we can keep this line ?

File: .cirrus/setup.sh [hunk 2]
```diff
@@ -2,26 +2,24 @@
...
 
-curl -o mpdev https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/scripts/dev
-chmod +x mpdev
-mkdir -p ${BASE_FOLDER}/bin
-mv mpdev ${BASE_FOLDER}/bin
+gcloud components install gke-gcloud-auth-plugin kubectl --quiet
```
Comment (jCOTINEAU) [inline .cirrus/setup.sh:19]: if we know install kubectl with the cloud plugin, we can remove the manual download from the Dockerfile?

File: .cirrus/tasks.yml [hunk 3]
```diff
@@ -92,13 +92,15 @@ verify_gcp_app_template: &VERIFY_GCP_APP_TEMPLATE
   only_if: $TRIGGER == "PUSH_STAGING_DEPLOYER" || $BRANCH == "master"
   env:
     BASE_FOLDER: "/root/.gcp/cache"
-    PATH: "${BASE_FOLDER}/bin:${BASE_FOLDER}/google-cloud-sdk/bin:${PATH}"
+    PATH: "${BASE_FOLDER}:${BASE_FOLDER}/google-cloud-sdk/bin:${PATH}"
```
Comment (jCOTINEAU) [inline .cirrus/tasks.yml:95]: i struggle understanding why we need this PATH variable at all considering we already modify it in the docker image ?

File: .cirrus/tasks.yml [hunk 4]
```diff
@@ -92,13 +92,15 @@ verify_gcp_app_template: &VERIFY_GCP_APP_TEMPLATE
...
     KUBE_CONFIG: "/tmp/kubeconfig"
     KUBECONFIG: "${KUBE_CONFIG}"
     HOME: "/tmp"
+    GCLOUD_CLI_VERSION: 495.0.0
+    MPDEV_VERSION: 0.12.4
   gcp_cache:
     folder: ${BASE_FOLDER}
-    fingerprint_script: grep "gcloud" .tool-versions | awk '{print $NF}'
+    fingerprint_script: echo "${GCLOUD_CLI_VERSION}+${MPDEV_VERSION}"
```
Comment (jCOTINEAU) [inline .cirrus/tasks.yml:103]: now that the install of gcloud-cli is made in the dockerfile we don't even need the cache, we can entirely remove this feature.

**Review by jCOTINEAU — CHANGES_REQUESTED** [review]

File: .cirrus/setup.sh (hunk 2 above)
Comment (carminevassallo) [reply .cirrus/setup.sh:19]: We don't install it in the VM. The dockerfile in this repo will be used in the other template.

File: .cirrus/tasks.yml (hunk 3 above)
Comment (carminevassallo) [reply .cirrus/tasks.yml:95]: As said above, we don't use the dockerfile here.

File: .cirrus/tasks.yml (hunk 4 above)
Comment (carminevassallo) [reply .cirrus/tasks.yml:103]: see my previous comment

**Review by davividal — APPROVED** [review]
LGTM!

**Review by jCOTINEAU — APPROVED** [review]
lgtm.

**Pull request merged**